exercises/robot-simulator/defs.go

```diff
@@ -19,14 +19,14 @@ type Command byte // valid values are 'R', 'L', 'A'
 type RU int
 type Pos struct{ Easting, Northing RU }
 type Rect struct{ Min, Max Pos }
-type DirAt struct {
+type Robot struct {
 	Dir
 	Pos
 }
 
 // additional definition used in step 3
 
-type Place struct {
+type Robot3 struct {
```

Comment:
Hm. Yeah, `Place` isn't great. `Robot3` has the disadvantage that when reading it we can't tell how it's distinct from `Robot` or `Robot2`, though.

`NamedRobot` is missing the idea that it's also located somewhere. This is really tricky!